### DISCUSSION TITLE
Окончание поддержки виджета 2.0

### DISCUSSION THREAD

**Discussion by vermorag** [cdek-it/widget#43] [Announcements]
С 01.04.2024 с widget.cdek.ru будет отключена поддержка Виджета 2.0 в варианте установки "Минимальный". 
Будут удален исходный код с серверов CDEK и установить его можно будет только скопировав на ваш сервер.

### UPD от 05.04.2024
Продлили переходный период для существующих клиентов по отказу от минимального варианта установки до 25.04.2024

### UPD от 07.11.2024
С 04.02.2025 мы закрываем устаревший незадокументированный метод автокомплита в Интеграции /city/getListByTerm/json.php, который использует 2 версия виджета.

**Весь функционал подбора города по названию станет недоступен.**

**Comment by Polearmik** [comment]
А новые ПВЗ тоже не будут добавляться в Виджет 2.0?

**Reply by vermorag** [reply]
Виджет 2.0 использует API CDEK 1.5 для получения списка ПВЗ. Информация об офисах в нем актуальна и при появления нового ПВЗ - он появится на карте.
Однако версия 1.5 объявлена устаревшей, новые клиенты на него подключаться не будут и, весьма вероятно, в ближайшее время эта версия API тоже может быть отключена.

**Comment by Andrew100500** [comment]
Вы сначала 4 версию рабочую сделайте, а потом отменяйте 2 версию.  Потому что ваша 3 версия - состоит из одних ошибок.

**Comment by Polearmik** [comment]
Не загружается Виджет 2.0 (исходный код загружен на свой сервер), получается он полностью перестал работать?
Машинка просто едет бесконечно...

**Comment by Tishka-Gamer** [comment]
Версия 2.5 тоже не поддерживается теперь?
